packages/contracts/contracts/test/ERC1271Mock.sol

```diff
@@ -0,0 +1,9 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity 0.8.10;
+
+contract ERC1271Mock {
+    function isValidSignature(bytes32, bytes memory) public pure returns (bytes4) {
+        return 0x41424344;
```

Comment:
should we do an actual `ECDSA.recover()` for the deployer address of this mock? 
so we can have more tests?
WDYT?